packages/plugin-legacy/src/index.ts

```diff
@@ -122,7 +123,11 @@ const _require = createRequire(import.meta.url)
 
 function viteLegacyPlugin(options: Options = {}): Plugin[] {
   let config: ResolvedConfig
-  const targets = options.targets || 'defaults'
+  const targets =
+    options.targets ||
+    browserslist() ||
```

Comment:
`browserslist` function always returns a value, so our default will never be used. I think we can use the `loadConfig` function instead.

`import { loadConfig as browserslistLoadConfig } from 'browserslist'`
```suggestion
    browserslistLoadConfig({ path: config.root }) || // we need to run this after the ResolvedConfig hook
```